src/chonkie/handshakes/turbopuffer.py

```diff
@@ -169,10 +149,11 @@ def search(
 
         # Use include_attributes to request extra fields
         results = self.namespace.query(
-            rank_by=("vector", "ANN", embedding),
+            rank_by=("vector", "ANN", embedding),  # type: ignore[arg-type]
             top_k=limit,
             include_attributes=["text", "start_index", "end_index", "token_count"],
         )
+        assert results.rows is not None
         return [
```

Comment:
although this works but i would've loved if we used this instead
```suggestion
        return [ # type:ignore[not-iterable]
```

Reply:
The assert is a type guard, it's more robust than ignoring the error since you wouldn't then ever get a "NoneType is not iterable" (unless running Python with `-O`, which erases the assert).